lib/jsonapi/matchers/shared.rb

```diff
@@ -0,0 +1,21 @@
+module Jsonapi
+  module Matchers
+    module Shared
+      def normalize_target(target)
+        if target.is_a?(::ActionController::TestResponse)
+          begin
+            JSON.parse(target.body)
+          rescue => e
+            @failure_message = "Expected response to be json string but was #{target.body.inspect}"
```

Comment:
Maybe. But I think the only error that can happen here is a deserialization error and that should be expressed by the message currently in place.

I will go ahead and add it because its super easy and certainly can't hurt.